src/api.ts

```diff
@@ -1,14 +1,48 @@
 import express from 'express';
 import snapshot from '@snapshot-labs/strategies';
 import scores, { blockNumByNetwork } from './scores';
-import { clone, sha256 } from './utils';
+import { clone, sha256, tsToBlockNum } from './utils';
+import client from './redis';
 
 const router = express.Router();
 
 router.get('/', (req, res) => {
   res.json(blockNumByNetwork);
 });
 
+router.get('/block', async (req, res) => {
+  let networks: any = req.query.network || '1';
+  const ts = req.query.ts || 1640300930;
```

Comment:
Shouldn't this be more like:

```suggestion
  const ts = req.query.ts || (Date.now() / 1e3);
```